src/components/ORAS_CLI/index.jsx

```diff
@@ -20,7 +20,7 @@ const os = [
     {
         label: 'Linux',
         value: 'linux',
-        content: 'VERSION=\"1.0.0\"\ncurl -LO \"https://github.com/oras-project/oras/releases/download/v${VERSION}/oras_${VERSION}_linux_amd64.tar.gz\"\nmkdir -p oras-install/\ntar -zxf oras_${VERSION}_*.tar.gz -C oras-install/\nsudo mv oras-install/oras /usr/local/bin/\nrm -rf oras_${VERSION}_*.tar.gz oras-install/',
+        content: 'VERSION=\"1.1.0\"\ncurl -LO \"https://github.com/oras-project/oras/releases/download/v${VERSION}/oras_${VERSION}_linux_amd64.tar.gz\"\nmkdir -p oras-install/\ntar -zxf oras_${VERSION}_*.tar.gz -C oras-install/\nsudo mv oras-install/oras /usr/local/bin/\nrm -rf oras_${VERSION}_*.tar.gz oras-install/',

```

Comment:
I am curious which ORAS web page will be reflected by this version change? The installation guide is already v1.1.0: https://oras.land/docs/installation/
cc @asmitbm

Reply:
This will change the version number in the CLI component on the homepage under Linux tab.

![Screenshot (739)](https://github.com/oras-project/oras-www/assets/81866614/154056a5-5707-4822-95d3-1e8bd57cf8bc)